utils/get-us-data.R

```diff
@@ -30,12 +30,20 @@ get_us_deaths <- function(data = "daily"){
      # De-cumulate to daily
        dplyr::group_by(state) %>% 
        dplyr::mutate(deaths = c(0, diff(deaths)),
-                     deaths = replace(deaths, deaths < 0 , 0)) %>% 
-       dplyr::ungroup() 
-   # Save daily deaths in all states
-    saveRDS(daily, here::here("data", "deaths-data-daily.rds"))
-    
-    return(daily)
+                     deaths = replace(deaths, deaths < 0 , 0),
```

Comment:
this is all good but I assumed it would be used as call to your nice anomalies function (which we should add to covidregionaldata at some point (no rush but perhaps issue with code)).